nikola/plugins/task/indexes.py

```diff
@@ -92,7 +92,8 @@ def get_path(self, classification, lang, dest_type='page'):
         """Return a path for the given classification."""
         if dest_type == 'rss':
             return [self.site.config['RSS_PATH'](lang)], True
```

Comment:
Please change the `True` to `'auto'` (I apparently forgot to change this a long time ago).